### PR TITLE
Update libvirt branch to hostos-devel

### PR DESCRIPTION
**Pull request by olavphibm**

### CODE REVIEW

**Comment by olavphibm** [comment]
test_plz

**Comment by sathnaga** [comment]
@olavphibm @maurorodrigues Looks like CI is failing, could you pls check, Tnx in advance :-)

**Comment by olavphibm** [comment]
test_plz

**Comment by sathnaga** [comment]
@olavphibm Build CI failed, pls check

File: libvirt/libvirt.yaml [hunk 1]
```diff
@@ -2,10 +2,9 @@ Package:
  name: 'libvirt'
  clone_url: 'https://github.com/open-power-host-os/libvirt.git'
  branch: 'powerkvm-v3.1.1'
```
Comment (maurorodrigues) [inline libvirt/libvirt.yaml]: shouldn't it  be hostos-devel?

Comment (olavphibm) [inline libvirt/libvirt.yaml]: Yeah, it should.

**Comment by olavphibm** [comment]
test_plz

**Comment by olavphibm** [comment]
Thanks, Mauro.
I thought this major update would require deeper changes in the spec file, so I was trying to use the updated (upstream) spec file as a base.

**Comment by olavphibm** [comment]
test_plz

**Comment by olavphibm** [comment]
test_plz

**Comment by olavphibm** [comment]
test_plz

**Comment by olavphibm** [comment]
test_plz

**Comment by sathnaga** [comment]
@olavphibm @maurorodrigues Build CI is failing, pls check

**Comment by olavphibm** [comment]
test_plz

**Comment by olavphibm** [comment]
test_plz

**Comment by olavphibm** [comment]
@sathnaga @maurorodrigues  Build succeeded in job 94.
Apart from fixing the spec file, we were having disk space issues in the builds system.

**Comment by maurorodrigues** [comment]
great! And unfortunately that's the side effect of not having many systems available for testing and development. Thanks for solving it @olavphibm

**Comment by sathnaga** [comment]
@olavphibm Thanks for fixing it, @maurorodrigues have reviewed disk space now  and cleaned from my side too, lets work it out permanently.

**Comment by sathnaga** [comment]
@olavphibm @maurorodrigues Able to update fine and run tests

**Comment by olavphibm** [comment]
@sathnaga Good, already merged it.

**Pull request merged**